Iterable/iterable-android-sdk
# Anonymous Track Update User

## Description

- **Release Notes**
> We have successfully implemented functionality for tracking Anonymous user update events.

> We've modified the document for Anonymous user update tracking.

- **Test Steps**
> Trigger update user event without logging into the SDK. If the criteria are not matched, It will store the event data in local storage or modified data if already exists. If matched, then it will create a user with your UUID. Using that UUID, you can search users over the iterable dashboard. Here you can see the event list in history.
